app/javascript/src/components/Dashboard/Contacts/Pane/Form.jsx

```diff
@@ -0,0 +1,108 @@
+import React, { useState, useContext } from "react";
+
+import { Formik, Form } from "formik";
+import { Toastr, Button, Pane } from "neetoui/v2";
+import { Input, Select } from "neetoui/v2/formik";
+
+import formValidationSchemas from "constants/formValidationSchemas";
+import { DashboardContext } from "contexts/dashboard";
+
+import { MONTHS, CONTACTS, FORM_ROLE_DROPDOWN } from "../constants";
+import { nameAndRole } from "../utils";
+
+const ContactForm = ({ contact, isEdit }) => {
+  const [submitted, setSubmitted] = useState(false);
+  const { setShowNewContactPane } = useContext(DashboardContext);
+
+  const handleSubmit = values => {
+    const date = new Date();
+
+    const createdAt = `${
+      MONTHS[date.getMonth()]
+    }, ${date.getDate()}, ${date.getFullYear()}`;
+
+    CONTACTS.unshift({
+      ...values,
+      name_role: nameAndRole({
+        name: `${values.firstName} ${values.lastName}`,
+        role: values.role.label,
+      }),
+      created_at: createdAt,
+    });
+
+    setShowNewContactPane(false);
+    Toastr.success("Successfully created contact.");
+  };
+
+  return (
+    <Formik
+      initialValues={contact}
+      onSubmit={handleSubmit}
+      validateOnBlur={submitted}
+      validateOnChange={submitted}
+      validationSchema={formValidationSchemas.contactsForm}
```

Comment:
Please make changes to `validationSchema` and `initialValues` as mentioned earlier.